app/controllers/pages_controller.rb

```diff
@@ -1,5 +1,5 @@
 class PagesController < ApplicationController
-  skip_before_action :authenticate_user!, only: [ :home ]
+  skip_before_action :authenticate_user!, only: [:home, :index, :show]
```

Comment:
its probably not really an issue, but i m a bit confused about the index and show here bc the pages controller doesnt have those actions, so i kinda thought we wouldnt need to put them here?

Reply:
a yeah, probably not. Obviously, I followed the kit tutorial so I accidentally left that in PagesController.